am-lib/src/main/java/uk/gov/hmcts/reform/amlib/models/ExplicitAccessGrant.java

```diff
@@ -0,0 +1,25 @@
+package uk.gov.hmcts.reform.amlib.models;
+
+import com.fasterxml.jackson.core.JsonPointer;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import uk.gov.hmcts.reform.amlib.enums.Permission;
+
+import java.util.Map;
+import java.util.Set;
+
+@Data
+@Builder
+@AllArgsConstructor
+public class ExplicitAccessGrant {
+
+    private final String resourceId;
+    private final String accessorId;
+    private final String accessType;
+    private final String serviceName;
+    private final String resourceType;
+    private final String resourceName;
+    private final Map<JsonPointer, Set<Permission>> attributePermissions;
+    private final String securityClassification;
```

Comment:
We should use SecurityClasification enum here to make it less error prone. Not sure if this is the right story though.

Reply:
I think there is a story for that in the backlog. It just hasn't been refined enough to play. So will leave it for now

Reply:
Ok, let's leave it for now as is.

Reply:
Also seen it is part of the next PI